game/ato/flightplans/waypointbuilder.py

```diff
@@ -304,6 +304,13 @@ def oca_strike_area(self, target: MissionTarget) -> FlightWaypoint:
         return self._target_area(f"ATTACK {target.name}", target, flyover=True)
 
     def assault_area(self, target: MissionTarget) -> FlightWaypoint:
+        """A destination waypoint used by air-assault ground troops.
+
+        This waypoint is an implementation detail for CTLD and should not be followed by
+        aircraft.
+        """
+        # TODO: Add a property that can hide this waypoint from the player's flight
+        # plan.
```

Comment:
I personally think it is great to still have that waypoint in the flight plan so that i know where the "hot zone" is. Also it is draggable via the Liberation UI so that i can fine tune exactly where the troops are going.. This is sometimes needed when the DCS logic for the capture event is just bad.

Reply:
Adding points of interest to the kneeboard (and the map for some aircraft) would be cool, but if it's part of the flight plan it will affect TOT planning, fuel estimation, escort pruning, etc.